src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$main-aqua-color: #00acdc;
+$main-dark-color: #060b35;
+$main-grey-color: #616070;
+$card-width: 200px;
+$right-card-margin: 48px;
+$max-main-width: $card-width * 4 + $right-card-margin * 3 + 80px;
```

Comment:
I think that extra
Mb you can pass test when you delete it
```suggestion
$max-main-width: $card-width * 4 + $right-card-margin * 3;
```

Reply:
![image](https://user-images.githubusercontent.com/113321536/222684782-c6e97273-f5af-4ed4-9671-cb95706efaa8.png)
without this 80px